book/content/make/make.md

```diff
@@ -694,6 +705,110 @@ The published Makefile in the repository does not contain the paper, but this
+
+The reproducible paper linked above shows one way of doing it:
+After the results are computed, they are written out in the form of a LaTeX
+table.
+Here is how one of these tables looks like right after it was computed:
+
+```latex
+\begin{tabular}{lrrr|rrrr}
+Property & HypoParsr & Sniffer & Suitability & Pattern & Type & No Tie & Full\\
+\hline
+Delimiter & 87.48 & 86.82 & 65.41 & 92.61 & 88.33 & 91.38 & \textbf{94.92}\\
+Quotechar & 82.90 & 92.36 & 44.60 & 95.23 & 90.10 & 93.80 & \textbf{97.36}\\
+Escapechar & 87.96 & 94.37 & 74.85 & 97.95 & 96.26 & 95.44 & \textbf{99.25}\\
+Overall & 80.60 & 85.45 & 38.19 & 90.99 & 83.61 & 90.61 & \textbf{93.75}\\
+\hline
+```
+
+To include this table into your manuscript, you can use LaTeX's `\input{}`
+function. If the file with the table is called `mytable.tex`, this command
+can insert it into your manuscript:
+
+```latex
+\begin{table}
+\input{mytable}
+\end{table}
+```
+
+An alternative is to make use of variables.
+Instead of creating a table in a separate file, you can write a table skeleton
+and populate it with variables.
+The results you compute are associated with the variables, and once your
+manuscript is compiled, variables are exchanged for real numerical results.
+Here is how such a table looks like in your manuscript:
+
+```latex
+\begin{table}
+    \begin{tabular*}{ccc}
+        \textbf{Variable} & \textbf{Mean}   & \textbf{Std. deviation} \
+        \hline
+        Variable 1        & \var1mean       & \var1std                \
+        Variable 2        & \var2mean       & \var2std                \
+    \end{tabular*}
+\end{table}
+```
+
+Ỳou may notice that `\var1mean` is no standard LaTeX command: It is a variable
+that you can define yourself!
+How is this done?
+Have your script print the results you compute within a `\newcommand{}{}`
+definition into a file, for example like this (simplified Python example):
+
+```python
+# this loops to data vectors of two variables (data1, data2), compute the
+# mean and standard deviation, and print the results together with the
+# variable name ('var1', 'var2')
+for name, data in (['var1', data1], ['var2', data2]):
+    mean = np.mean(data)
+    std = np.mean(data)
+    print('\\newcommand{\\%s }{ %f }' % (name + 'mean', mean))
+    print('\\newcommand{\\%s }{ %f }' % (name + 'std', std))
+```
+
+Let's say the mean of the first dataset is 9.2, the definition would look like
+this: `\newcommand{\var1mean}{9.2}`.
+Note that this example uses Python, but you can use any language or method you
+are familiar with to print definitions like this.
+With this definition, LaTeX exchanges the table cell with `\var1mean` with the
+numeric result from the computation.
+You can capture the definitions and write them to a file using redirection with
+`>`.
+In this example, we write it to a file called `results_def.tex`
+
+```makefile
+results_def.tex: code/make_summary_stats.py
+    python code/make_summary_stats.py > results_def.tex
```

Comment:
Judging by [the preview](https://deploy-preview-915--the-turing-way.netlify.com/make/make.html#including-numerical-results-and-tables) it seems that the syntax highlighting doesn't work with spaces before the command. Using a tab should fix that.

Reply:
I've pushed a change and will check the preview :+1: Thanks for being so attentive!

Reply:
Indeed, looks good now. Thanks again for the attentive review :)

Reply:
Glad I could help! Thanks for the contribution!